package.json

```diff
@@ -119,6 +119,7 @@
     "eslint-plugin-ignore-erb": "^0.1.1",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.3.1",
+    "eslint-plugin-no-only-tests": "^2.4.0",
```

Comment:
took me a minute to understand what this was for (I didn't understand what .only did in this context) 

> Designed to prevent you from committing .only tests to CI, disabling tests for your whole team.

Makes sense (we don't want to accidentally do that).